acceptancetests/assess_persistent_storage.py

```diff
@@ -223,6 +222,63 @@ def assess_charm_removal_single_block_and_filesystem_storage(client):
     else:
         log.info('Detach single block device storage output check: {}'.format(
             detach_single_blk_output_check))
+    # storage status change after remove-application takes some time.
+    # from experiments even 30 seconds is not enough.
+    time.sleep(60)
+    # check the real status of storage after remove-application
+    storage_output = json.loads(
+        client.get_juju_output('storage', '--format', 'json', include_e=False))
+    storage_list = storage_output['storage'].keys()
+    if len(storage_list) != 1:
+        log.error('\n'.join(storage_list))
+        raise JujuAssertionError(
+            'Unexpected number of storage unit(s). '\
+            'Found: {}\nExpected: 1\ntest terminated.'.format(
+            str(len(storage_list))))
+    elif single_fs_id in storage_list:
```

Comment:
Why are these else-ifs? Aren't they all valid? I realize it will work in the happy case, and you are raising errors, so if any of them trigger they will stop. However, it reads poorly and is prone to error if you change the logic conditions found in any of the elifs. Can you instead break these out as separate asserts instead of the messy if block?